model/jpa/src/main/resources/META-INF/jpa-changelog-26.0.0.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!--
+  ~ * Copyright 2024 Red Hat, Inc. and/or its affiliates
+  ~ * and other contributors as indicated by the @author tags.
+  ~ *
+  ~ * Licensed under the Apache License, Version 2.0 (the "License");
+  ~ * you may not use this file except in compliance with the License.
+  ~ * You may obtain a copy of the License at
+  ~ *
+  ~ * http://www.apache.org/licenses/LICENSE-2.0
+  ~ *
+  ~ * Unless required by applicable law or agreed to in writing, software
+  ~ * distributed under the License is distributed on an "AS IS" BASIS,
+  ~ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ * See the License for the specific language governing permissions and
+  ~ * limitations under the License.
+  -->
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+    <changeSet author="keycloak" id="26.0.0-org-alias">
+        <addColumn tableName="ORG">
+            <column name="ALIAS" type="VARCHAR(255)">
+                <constraints nullable="false"/>
```

Comment:
Not sure if we need to care about migration as the feature is tech preview?

Reply:
We do need to add it as separate changelog anyway.

Reply:
Updates will fail. Do we want migration?

Reply:
My personal view on this is that we should have a migration. :)

Reply:
@vramik @sguilhen Can we use `valueComputed` together with `update` in Liquibase to update the new column with the value of the `NAME` column?

Reply:
Migration added.